### PR TITLE
Update pythonpackage.yml

### PR DESCRIPTION
**Pull request by vaaaaanquish**
add GitHub Actions CI/CD

### CODE REVIEW

**Comment by hirosassa** [comment]
I'm currently working on `gokart`!!

**Comment by 6syun9** [comment]
@hirosassa How do you configure `yapf`? yapf doesn't work...

**Comment by hirosassa** [comment]
Still working on `yapf` ... it does't work...

**Comment by 6syun9** [comment]
@hirosassa We must set`--exclude=.eggs/* ` .  

please look this one.
https://github.com/m3dev/thunderbolt/pull/3/checks?check_run_id=222710435#step:5:1
```
--- ./.eggs/setuptools_scm-3.3.3-py3.7.egg/setuptools_scm/hg.py	(original)
+++ ./.eggs/setuptools_scm-3.3.3-py3.7.egg/setuptools_scm/hg.py	(reformatted)
@@ -15,9 +15,7 @@
...
```

**Comment by 6syun9** [comment]
@hirosassa @nishiba Please check PR! We can turn off travis-ci!

**Comment by hirosassa** [comment]
Oh!!!!! That's GREAT!!!!!

**Comment by hirosassa** [comment]
LGTM!

File: .github/workflows/pythonpackage.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - name: Lint with yapf
+      run: |
+        pip install yapf
+        diffs=$(yapf -dr --style=yapf.ini --exclude=examples/* --exclude=.eggs/* ./)
+        if [ -n "$diffs" ]; then	
+          echo "failed yapf"	
+          echo $diffs	
+          exit 1	
+        fi	
+        echo "pass yapf"
```
Comment (hirosassa) [inline .github/workflows/pythonpackage.yml]: [nits] found unnecessary white spaces

**Pull request merged**